community/contributing/docs/github-changes.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: single
+title:  "Make a Change Using the GitHub UI"
+sidebar:
+  nav: community
+---
+
+{% include toc %}
+
+# Open a pull request on GitHub
+
+1. On the page where you see the issue, select the pencil icon at the top right. You can also scroll to the bottom of the page and select **Edit this page**.
```

Comment:
This is on the markdown file correct? When first reading this, I thought you were talking about approaching this from spinnaker.io cause there are buttons like Suggest an edit etc on each documentation page https://s.armory.io/eDu6QX5N

So I think it'd be good to clarify where we're starting. Maybe have step 1 be `1. Navigate to the Spinnaker documentation repository[https://github.com/spinnaker/spinnaker.github.io]

Reply:
Done! I've made this change and it's now a new commit.